### ISSUE TITLE
[BUG] Contract address is treated as invalid in send stx window

### ISSUE THREAD

**Issue by LNow** [blockstack/blockstack-app#2514]
I can't send STX directly to contract address using built in `Send` functionality, because all contract addresses are treated as invalid.

![image](https://user-images.githubusercontent.com/829210/176213386-f649bee0-eefe-4386-a46a-2161af2c3116.png)

For me it is a bug, but maybe you decided to not allow direct transfers to contract addresses due to some reason.

**Comment by markmhendrickson** [comment]
If you take off the `.pox`, does it work as intended?

**Comment by LNow** [comment]
@markmhx yes. And just to clarify it doesn't work with any contract, not only the `.pox`.
